README.md

```diff
@@ -381,6 +387,52 @@ Sometimes you may want to use your own serialization for some objects and do not
+- config.setProperty("hazelcast.client.invocation.retry.pause.millis", "500");
+The default retry wait time is 1000 msecs(i.e. 1 second).
+
+Of course, this retry does not continue indefinitely but finishes when the invocation timeout occurs. This duration is configurable with this configuration property:
+- config.setProperty("hazelcast.client.invocation.timeout.seconds", "30");
+The default timeout is 120 seconds.
+
+## Client Backpressure
+Sometimes, you may want to slow down the client operations to the cluster (e.g. your servers are overloaded). Then the client can be configured to wait until number of outstanding invocations whose reponses are not received to become less than a certain number. This is called "Client Backpressure". By default, the backpressure is disabled. There are a few properties which control the backpressure. These client configuration properties are:
+- "hazelcast.client.max.concurrent.invocations" : The maximum number of concurrent invocations allowed. To prevent the system from overloading, user can apply a constraint on the number of concurrent invocations. If the maximum number of concurrent invocations has been exceeded and a new invocation comes in, then hazelcast will throw HazelcastOverloadException. By default this property is configured as INT32_MAX.
+- "hazelcast.client.invocation.backoff.timeout.millis": Controls the maximum timeout in millis to wait for an invocation space to be available. If an invocation can't be made because there are too many pending invocations, then an exponential backoff is done to give the system time to deal with the backlog of invocations. This property controls how long an invocation is allowed to wait before getting a <code>HazelcastOverloadException</code>. When set to -1 then <code>HazelcastOverloadException</code> is thrown immediately without any waiting. This is the default value.
+
+# Client Connection Strategy
+Hazelcast client cluster connection and reconnection strategy can be configured. Sometimes, you may not want your application to wait for the client to connect to the cluster, you may just want to get the client and let the client connect in the background. This is configured by:
+- ClientConfig::getConnectionStrategyConfig().setAsyncStart(bool);
+
+When this config is set true, the client creation won't wait to connect to cluster. The client instance will throw exception for any request, until it connects to cluster and become ready.
+If it is set to false (the default case), <code>HazelcastClient(const ClientConfig)</code> will block until a cluster connection established and it's ready to use client instance.
+
+## Configure Client Reconnect Strategy
+You can configure how the client should act when the client disconnects from the cluster for any reason. This is configured using the following configuration:
+- ClientConfig::getConnectionStrategyConfig().setReconnectMode(const hazelcast::client::config::ClientConnectionStrategyConfig::ReconnectMode &);
```

Comment:
Instead of bulleted list:

```
ClientConfig::getConnectionStrategyConfig().setReconnectMode(const hazelcast::client::config::ClientConnectionStrategyConfig::ReconnectMode &);
```